test/README.external

```diff
@@ -67,3 +71,30 @@ within the OpenSSL source code.
 The community is encouraged to contribute patches which reduce the number of
 suppressions that are currently present.
 
+
+Python PYCA/Cryptography test suite
+===================================
+
+This python test suite runs cryptographic tests with a local OpenSSL build as
+the implementation.
+
+First checkout the PYCA/Cryptography module into ./pyca-cryptography using:
+
+$ git submodule update --init
+
+Then configure/build OpenSSL compatible with the python module:
+
+$ ./config shared enable-external-tests
+$ make
+
+The tests will run in a python virtual environment which requires virtualenv
+to be installed.
+
+$ HARNESS_VERBOSE=1 TESTS="test_external" make test
```

Comment:
There is also the manner that's supported by every `make` implementation in existence:

```
$ make test TESTS=test_external_pyca VERBOSE=1
```

`VERBOSE`, as well as `V`, is recognised by `test/run_tests.pl` which will set `HARNESS_VERBOSE` accordingly.